.rubocop.yml

```diff
@@ -9,3 +9,8 @@ Style/TrailingCommaInArrayLiteral:
 
 Style/TrailingCommaInHashLiteral:
   Enabled: false
+
+Naming/FileName:
+  Enabled: true
+  Exclude:
+    - lib/kubernetes-deploy.rb
```

Comment:
Should we just fix the file name?

Reply:
Ya, I think it needs to match for ultimately silly reasons.